src/ophys_etl/transforms/classification.py

```diff
@@ -143,6 +147,42 @@ class InferenceInputSchema(ArgSchema):
         required=True,
         description="Filepath to dump json output."
     )
+    model_registry_table_name = fields.Str(
+        required=False,
+        missing="ROIClassifierRegistry",
+        description=("The name of the DynamoDB table containing "
+                     "the ROI classifier model registry.")
+    )
+    model_registry_env = fields.Str(
+        required=False,
+        missing="prod",
+        description=("Which environment to query when searching for a "
+                     "classifier model path from the classifier model "
+                     "registry. Possible options are: ['dev', 'stage', 'prod]")
+    )
+
+    @pre_load
+    def determine_classifier_model_path(self, data: dict, **kwargs) -> dict:
+        if "classifier_model_path" not in data:
+            # Can't rely on field `missing` param as it doesn't get filled in
+            # until deserialization/validation. The get defaults should match
+            # the 'missing' param for the model_registry_table_name and
+            # model_registry_env fields.
+            table_name = data.get("model_registry_table_name",
+                                  "ROIClassifierRegistry")
+            model_env = data.get("model_registry_env", "prod")
+            model_registry = RegistryConnection(table_name=table_name)
+            model_path = model_registry.get_active_model(env=model_env)
+            data["classifier_model_path"] = model_path
+        return data
+
+    @validates("model_registry_env")
+    def validate_model_registry_env(self, env: str, **kwargs):
```

Comment:
FYI for the future: You can simplify this in the Field. There's a validate keyword argument, and you can do `validate=OneOf(...)`

https://marshmallow.readthedocs.io/en/stable/marshmallow.validate.html#marshmallow.validate.OneOf